doc/user_guide/psy_data.rst

```diff
@@ -80,3 +82,124 @@ the PSyData API, for example ``ProfileTrans``, ``GOceanExtractTrans``
 and ``LFRicExtractTrans``. A user can develop additional transformations
 and corresponding runtime libraries for additional functionality.
 Refer to :ref:`dev_guide:psy_data` for full details about the PSyData API.
+
+
+.. _psydata_read_validation:
+
+Read-Only Verification
+----------------------
+The PSyData interface is being used to verify that read-only variables
+in a kernel are not overwritten. The ``ReadOnlyVerifyTrans`` (in 
+``psyir.tranformations.read_only_verify_trans``) uses the dependency
```

Comment:
I added a generic link to the transformation section only, since the online reference guide doesn't have the read-only link section yet :( It's a bit of a catch 22 I guess - but at least this way we have a link that works, though it could be improved to point to the right class later.

Reply:
Oh yes, I always seem to forget about that problem. Please could you remove the full stop that has appeared after the parentheses enclosing the link.

Reply:
Done.